backend/video/views.py

```diff
@@ -27,9 +27,9 @@
     method="get",
     manual_parameters=[
         openapi.Parameter(
-            "video_url",
+            "file_url",
```

Comment:
Just a suggestion - `file_url` seems to give a sense as if it is a direct link to the file. Should we just call it `url` ?

Reply:
`url` seems very non-descriptive, imo. Maybe, 'multimedia_url'?